### ISSUE TITLE
Implement Feedback Interface in finesse-frontend

### ISSUE THREAD

**Issue by k-allagbe** [ai-cfia/finesse-frontend#75]
Create a user-friendly feedback interface for Finesse, allowing users to submit feedback, bug reports, or feature requests with ease. The interface should match our GitHub issue templates for automated issue creation and be accessible to all users.

Tasks:
- [ ] Design a feedback form that aligns with GitHub issue templates and accessibility standards.
- [ ] Implement form validation to ensure necessary information is captured.
- [ ] Develop a secure, intuitive UI for the feedback form, incorporating accessibility features.
- [ ] Test the feedback form for compatibility and responsiveness across different browsers and devices.

Acceptance Criteria:
- Intuitive and easy-to-use feedback interface.
- Form fields correspond with GitHub issue template fields.
- Fields for different feedback types (e.g., bug report, feature request).
- Validation and clear error messages.
- Responsive design, working across major browsers and devices.
- Feedback form submissions ready for backend GitHub issue integration.
- Compliance with accessibility standards.

**Comment by rngadam** [comment]
@k-allagbe since this is probably functionality that should cross frontends, we probably want to make this visibile to DevSecOps?

In terms of implementation, thinking of React component for feedback feedback-react and a blueprint feedback-backend (interacting with Github) that can be imported to the products.

Although have you considered just having the users click through Github? Think the main issue there is we don't want the users to have to create an account with Github, but if we integrate Github to Azure, we can use SSO login with our organizational account.

**Comment by k-allagbe** [comment]
> [@k-allagbe](https://github.com/k-allagbe) since this is probably functionality that should cross frontends, we probably want to make this visibile to DevSecOps?
> 
> In terms of implementation, thinking of React component for feedback feedback-react and a blueprint feedback-backend (interacting with Github) that can be imported to the products.
> 
> Although have you considered just having the users click through Github? Think the main issue there is we don't want the users to have to create an account with Github, but if we integrate Github to Azure, we can use SSO login with our organizational account.

Yes mainly to avoid having to create a github account. I added it to the DevSecOps project. @ThomasCardin, @SonOfLope please check it out and share your thoughts.

**Comment by SonOfLope** [comment]
> > [@k-allagbe](https://github.com/k-allagbe) since this is probably functionality that should cross frontends, we probably want to make this visibile to DevSecOps?
> > In terms of implementation, thinking of React component for feedback feedback-react and a blueprint feedback-backend (interacting with Github) that can be imported to the products.
> > Although have you considered just having the users click through Github? Think the main issue there is we don't want the users to have to create an account with Github, but if we integrate Github to Azure, we can use SSO login with our organizational account.
> 
> Yes mainly to avoid having to create a github account. I updated the description and added it to the DevSecOps project. @ThomasCardin, @SonOfLope please check it out and share your thoughts.

@k-allagbe we have to consider the end-users familiarity and comfort with such platform as Github. Our non-technical users may find GitHub confusing, which could deter them from providing feedback.

> In terms of implementation, thinking of React component for feedback feedback-react and a blueprint feedback-backend (interacting with Github) that can be imported to the products.

I think this solution is more viable to provide a more straightforward and familiar interface for the users without the need for a Github account. 

@ThomasCardin feel free to add your input if needed.

**Comment by ThomasCardin** [comment]
I agree with @SonOfLope , if the inspectors are familiar with Github, we could consider the GitHub/Azure solution. Otherwise, having a form directly in the frontend would likely be the best approach.

**Comment by SonOfLope** [comment]
I also want to add that this is a requirement to be part of a [alpha.canada.ca subdomain](https://alpha.canada.ca/en/instructions.html) : 
![image](https://github.com/ai-cfia/finesse-frontend/assets/73133692/7a77e391-8e6a-45aa-8e40-bd0c42044d58)
